PLEP-0007.rst

```diff
@@ -0,0 +1,364 @@
+===============================================
+PLEP-0007 – Structure of Top-Level Sub-Packages
+===============================================
+
+.. _`10.5281/zenodo.3774573`: https://doi.org/10.5281/zenodo.3774573
+
++-------------------+---------------------------------------------+
+| PLEP              | 7                                           |
++===================+=============================================+
+| author(s)         | | Erik T. Everson                           |
+|                   | | Tulasi Parashar                           |
+|                   | | Stephen Vincena                           |
+|                   | | Nicholas A. Murphy                        |
++-------------------+---------------------------------------------+
+| contact email     | eeverson@ucla.edu                           |
++-------------------+---------------------------------------------+
+| date created      | 2019-10-27                                  |
++-------------------+---------------------------------------------+
+| date last revised | 2020-05-01                                  |
++-------------------+---------------------------------------------+
+| type              | standard                                    |
++-------------------+---------------------------------------------+
+| status            | in preparation                              |
++-------------------+---------------------------------------------+
+| DOI               | `10.5281/zenodo.3774573`_                   |
+|                   |                                             |
++-------------------+---------------------------------------------+
+
+**Contents**
+
+* `Abstract`_
+* `Detailed Description`_
+
+  * `Why?`_
+  * `Top-Level Sub-Packages`_
+
+* `Extensible Packages are Exempt`_
+* `Implementation`_
+* `Issues, Pull Requests, and Branches`_
+* `Backward Compatibility`_
+* `Decision Rationale`_
+
+Abstract
+========
+
+This PLEP defines the top-level structure of ``PlasmaPy`` (i.e.
+``plasmapy.<subpackage>``).  Its intent is to name the top-level
+sub-packages and define the general scope of each package.
+Sub-package details are left to be defined in package focused PLEPs,
+which should *NOT* conflict with this PLEP.  Defining the top-level
+structure should:
+
+  #. keep the top-level namespace from getting bloated
+  #. help developers decided where new code should be placed
+  #. help navigate users to the functionality they need
+
+If new code does NOT fall within the current framework, then this PLEP
+needs to be modified/updated accordingly before any new top-level
+sub-packages are created.
```

Comment:
Or, presumably, the code should be ajusted to fit this framework?

Reply:
I've reworded this a bit to indicate that amending the PLEP should be the last resort option.  I assumed that would happen by default, since it will always be easier to conform code than to amend the PLEP and get it accepted.

This statement is here more so for the off chance that we overlooked something that should be a top-level package.